packages/data-point/lib/stores/store-manager.test.js

```diff
@@ -56,7 +54,7 @@ describe('add', () => {
     expect(error).toMatchSnapshot()
   })
 
-  test('It should throw error with information if item already exist', () => {
+  test('When ObjectStoreManager is only called once, it should return the object id and spec with no errors', () => {
```

Comment:
Only to follow the patter of `It should...`

`It should return the object id and spec with no errors when ObjectStoreManager is called once`